lxd/instance/drivers/driver_lxc.go

```diff
@@ -6032,12 +6069,12 @@ func (d *lxc) insertMountLXC(source, target, fstype string, flags int) error {
 	return nil
 }
 
-func (d *lxc) insertMount(source, target, fstype string, flags int, shiftfs bool) error {
-	if d.state.OS.LXCFeatures["mount_injection_file"] && !shiftfs {
+func (d *lxc) insertMount(source, target, fstype string, flags int, idmapped string) error {
+	if d.state.OS.LXCFeatures["mount_injection_file"] && idmapped == "" {
```

Comment:
@brauner yeah I agree about "none" being clearer than "", but I was referring to this line where the case for idmapped being "" is handled. I wondered in what scenario would imapped == "" rather than "none" and what is the difference? From looking further up in the code that it seems to be when a unix device is passed in, but I wondered why using "none" in those cases wouldn't be appropriate?

Reply:
I think this was a left-over.